README.md

```diff
@@ -109,7 +108,6 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 - [8 non-code ways to contribute to open source](https://opensource.com/life/16/1/8-ways-contribute-open-source-without-writing-code) by [OpenSource](https://twitter.com/OpenSourceWay)
 
 ## Using Version Control
-- [Think Like (a) Git](http://think-like-a-git.net/) - Git introduction for "advanced beginners," but are still struggling, in order to give you a simple strategy to safely experiment with git.
```

Comment:
Why was this removed? This appears to be just fine.

Reply:
@erictleung  yes i see know i was testing this in Firefox and i got The extension HTTPS everywhere and it made the stie look broken because its HTTP. i wil update this pull request